akka-actor-typed-tests/src/test/scala/akka/actor/typed/internal/adpater/PropsAdapterSpec.scala

```diff
@@ -18,7 +19,37 @@ class PropsAdapterSpec extends AnyWordSpec with Matchers {
     "default to akka.dispatch.SingleConsumerOnlyUnboundedMailbox" in {
       val props: Props = Props.empty
       val pa: actor.Props = PropsAdapter(() => Behaviors.empty, props, rethrowTypedFailure = false)
-      pa.mailbox shouldEqual "akka.actor.typed.default-mailbox"
+      pa.mailbox === ("akka.actor.typed.default-mailbox")
```

Comment:
Missing the `should` to actually test this:

```suggestion
      pa.mailbox should ===("akka.actor.typed.default-mailbox")
```